### ISSUE TITLE
Bug that causes tabs to not load

### ISSUE THREAD

**Issue by huizichin** [XFox111/TabsAsideExtension#64]
### Description
When you restore a list and set aside again quickly and this happens.
![image](https://user-images.githubusercontent.com/16850884/104853985-68bd9a00-593f-11eb-96dd-7fb6969dff88.png)

### Reproduction steps
1. Set tabs aside
2. Restore tabs
3. quickly set tab aside again
4. Restore tabs, and you should see the problem. It only happens sometimes

### Expected behavior
No error

### Environment
Windows 10 20H2
Edge 87.0.664.75

**Comment by XFox111** [comment]
Thanks for contributing to the project!
Yeah, I've managed to reproduce the bug. However, I don't know how long it will take to fix it since this scenario is quite uncommon and the bug can be considered as low priority

**Comment by AmineI** [comment]
I agree, it is uncommon, but it is what someone would probably do to achieve #66 ... I have a feeling that closing the pane on restore could alleviate the problem, but it's not a solution per se. I'll try to see what could help with this once 2.0 is released
